packages/reports/src/Constants/__tests__/data-table-constants.spec.tsx

```diff
@@ -0,0 +1,570 @@
+        render(stakeColumn.renderCellContent({ cell_value, is_footer: false }));
+
+        expect(screen.getByText(cell_value)).toBeInTheDocument();
+    });
+    it('should render cell content for "Sell time" column', () => {
+        const sellTimeColumn = columns[5];
+        const cell_value = '2023-05-30 13:34:56';
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(sellTimeColumn.renderCellContent({ cell_value, is_footer: false }));
+
+        expect(screen.getByText(`${cell_value} GMT`)).toBeInTheDocument();
+    });
+    it('should render cell content for "Contract value" column', () => {
+        const contractValueColumn = columns[6];
+        const cell_value = '2000';
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(contractValueColumn.renderCellContent({ cell_value, is_footer: false }));
+
+        expect(screen.getByText(cell_value)).toBeInTheDocument();
+    });
+    it('should render cell content for "Total profit/loss" column', () => {
+        const totalProfitLossColumn = columns[7];
+        const cell_value = '500';
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(totalProfitLossColumn.renderCellContent({ cell_value }));
+
+        expect(screen.getByText(cell_value)).toBeInTheDocument();
+    });
+});
+
+describe('getOpenPositionsColumnsTemplate', () => {
+    const currency = 'USD';
+
+    it('should render all columns correctly', () => {
+        const columns = getOpenPositionsColumnsTemplate(currency);
+
+        columns.forEach(column => {
+            const { title } = column;
+            if (title) {
+                render(<div>{title}</div>);
+                expect(screen.getByText(title)).toBeInTheDocument();
+            }
+        });
+    });
+    it('should render cell content correctly for each column', () => {
+        const dummyRowObj = {
+            id: 'dummy_id',
+            contract_info: {
+                id: 'contract_info_id',
+                currency: 'USD',
+                profit: 100,
+            },
+            barrier: 1.2345,
+            is_sell_requested: false,
+            profit_loss: 50,
+        };
+
+        const columns = getOpenPositionsColumnsTemplate(currency);
+        columns.forEach(column => {
+            const { renderCellContent: content, col_index } = column;
+            if (content) {
+                let cell_value: React.ReactNode = '1000';
+                if (col_index === 'profit') cell_value = 50;
+                if (col_index === 'type') cell_value = 'Type';
+
+                const cellContent = content({
+                    row_obj: dummyRowObj,
+                    //@ts-expect-error type should be React.ReactNode
+                    cell_value,
```

Comment:
cell_value seems to be of string type. I think it should fix the issue if we remove the React.ReactNode type on it.

![image](https://github.com/binary-com/deriv-app/assets/56330681/fc0f1ce6-f056-41e5-8f6c-5df240f150ff)